packages/filesystem/src/node/node-filesystem.ts

```diff
@@ -77,7 +79,8 @@ export class FileSystemNode implements FileSystem {
             throw FileSystemError.FileIsDirectory(uri, 'Cannot resolve the content.');
         }
         const encoding = await this.doGetEncoding(options);
-        const content = await fs.readFile(FileUri.fsPath(_uri), { encoding });
+        const contentBuffer = await fs.readFile(FileUri.fsPath(_uri));
+        const content = iconv.decode(contentBuffer, encoding);
```

Comment:
wonder here as well, what `iconv` does what nodejs cannot with passed encoding. It seemed to work before

Reply:
see above.